### ISSUE TITLE
Replace image aria-label with alt and remove redundant aria-hidden

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-hotgraphic#356]
### Subject of the enhancement

To align with core change of using `alt` instead of `aria-label` - https://github.com/adaptlearning/adapt-contrib-core/issues/822

Testing confirmed all major screen readers handle `alt` and `aria-label` equivalently and Adapt would benefit from the broader benefits of `alt`. Benefits include:
-  alternative text displayed when image fails to load 
- alternative text retained when copying images into Word or similar tools.

Testing also confirmed an empty alt tag (`alt=""`) is widely supported by screen readers / browsers to ignore an image without the need for `aria-hidden`.

[Applies to image, lines 27-28](https://github.com/adaptlearning/adapt-contrib-hotgraphic/blob/7a38f0d25afb450a663942efbcb57ef2d5ce20bf/templates/hotgraphicLayoutPins.jsx#L27-L28).
